config.toml

```diff
@@ -171,6 +171,11 @@ githubbranch = "v1.0-branch"
   githubbranch = "v0.7-branch"
   url = "https://v0-7.kubeflow.org"
 
+[[params.versions]]
+  version = "v1.0"
+  githubbranch = "v1.0-branch"
+  url = "https://v1-0.kubeflow.org"
```

Comment:
This URL doesn't appear to be live at. What does the release guide say about when this gets created?

Reply:
The release guide does not mention adding this, but the config.toml file says that here is where we add new versions to website drop-down. 

Given that we do not have the URL live yet, we need to to [this](https://github.com/kubeflow/kubeflow/blob/master/docs_dev/releasing.md#creating-and-publishing-a-website-branch-for-vxy) step first and then come and add this.

I'll delete this last commit, so we can merge and we'll do this step when we're ready.